game/state/icematch.js

```diff
@@ -0,0 +1,21 @@
+const Player = require('./player')
+
+class GameState {
+  constructor () {
+    this.players = {}
+  }
+
+  addPlayer (client) {
+    this.players[client.sessionId] = new Player(0, 0, 'file.png')
+  }
+
+  removePlayer (client) {
+    delete this.players[client.sessionId]
+  }
+
+  moveDirection (sessionId, plane, change) {
```

Comment:
rather than `plane`, maybe `dimension`? We're dealing with X-dimensions and Y dimensions, not the X plane and Y plane (since it's 2d)

Reply:
also, `movePlayer` probs makes more sense, just to be consistent with the rest of the API